webapi/src/main/scala/org/knora/webapi/slice/admin/api/GroupsEndpoints.scala

```diff
@@ -96,8 +96,17 @@ object GroupsRequests {
     status: Option[GroupStatus] = None,
     selfjoin: Option[GroupSelfJoin] = None,
   )
+
   object GroupUpdateRequest {
-    implicit val jsonCodec: JsonCodec[GroupUpdateRequest] = DeriveJsonCodec.gen[GroupUpdateRequest]
+    implicit val jsonCodec: JsonCodec[GroupUpdateRequest] = DeriveJsonCodec
+      .gen[GroupUpdateRequest]
+      .transformOrFail(
+        request =>
+          if (List(request.name, request.descriptions, request.status, request.selfjoin).flatten.isEmpty)
+            Left("At least one field must be present.")
+          else Right(request),
+        identity,
+      )
```

Comment:
**issue**: A serializer/deserializer contains validation logic for the value type

IMHO this behavior of a codec is surprising in a bad way. The product type `GroupUpdateRequest` allows for all combinations of params and that includes all being `None`. A serializer/deserializer should then also be able to produce a `GroupUpdateRequest` json reflecting the all `None` state.

I would rather like to see this done programatically in the rest service or in a smart constructor. A codec to me feels like the last place I would assume such logic exists.